www/src/pages/404.js

```diff
@@ -2,18 +2,21 @@ import React from "react"
 import Container from "../components/container"
 import Link from "../components/localized-link"
 import FooterLinks from "../components/shared/footer-links"
+import Main from "../components/main-content"
 
 export default function FourOhFour() {
   return (
     <>
```

Comment:
We don't need this fragment any more since everything is wrapped in a `<Main>` tag.